### ISSUE TITLE
cg.dll now required in main package?

### ISSUE THREAD

**Issue by macropoloxxi** [glscene/GLScene#27]
Hi Pavel!

Current version is 2.3, right? Because GitHub still shows 2.2 as the version, but when you run the Delphi IDE it says 2.3...

I think that starting on V2.2, some features in GLScene_RT.bpl started to require a library that previously was not necesssary for the basic package:

![image](https://github.com/glscene/GLScene/assets/16530111/41d0f3ea-3d1c-4677-8f6a-a5a9dc8804d8)

In the _setupDLL.cmd is described as an NVidia library:

echo Copying nVidia CG DLLs
rem win32
copy "%~dp0cg.dll" %SystemRoot%\SysWOW64\
copy "%~dp0cgGL.dll" %SystemRoot%\SysWOW64\
rem win64
copy "%~dp0cg.dll" %SystemRoot%\System32\
copy "%~dp0cgGL.dll" %SystemRoot%\System32\

In fact I don't have any NVidia GPU (this is a notebook running on Intel) and it started to require that dll. If you register it the GLScene based code works OK, but previously there wasn't any dependency on external libraries an now it is. Just to confirm: if I run the same program, but compiled the last year (with V2.0 of GLScene I believe) it doesn't require this external .dll

Is it the cg.dll striclty required or it appear some imported function that it has to map and in fact it could be non-dependant as before?

Regarding the GLTF files: are they working? If you try to import a the Cube.GLTF of the samples in the GLSViewerD you always get:

![image](https://github.com/glscene/GLScene/assets/16530111/fe53d4a5-15c7-4e4b-9f94-62e1454ca17d)

Many thanks in advance! Regards, Pablo.

**Comment by glscene** [comment]
Yes, the release was 2.2 and 2.3 will be later this year.  I don't aware why GLScene_RT requires cg.dll - see dependencies screenshot in attachment. May be a unit was included from cg package and you need to check it. As for glTF import file it needs to be upgraded and if you could try to improve the code then after pull request glTF/glb models would be used in our demos.  
Pavel
![Dependencies](https://github.com/glscene/GLScene/assets/66052716/1f84a922-1bfb-4ccb-93b1-c8f309f37edd)
